### ISSUE TITLE
Check with iOS that you are using same endpoint calls

### ISSUE THREAD

**Issue by markoaamunkajo** [ruuvi/com.ruuvi.station#1100]
Description:

Please verify that you are using same endpoint calls on same pages and features.

Is history being fetched for each sensor in parallel or serial, on-demand or is it fetched when history page is viewed?

iOS card: https://github.com/ruuvi/com.ruuvi.station.ios/issues/1627

**Comment by andreevdenis** [comment]
Full sync procedure:

1) Executing postponed POST request if any were scheduled
2) Checking subscription for user /subscription
3) Getting user settings form the cloud /settings
4) Getting latest measurements for sensors /sensor-dense
5) Getting history per sensor in parallel via /get (if subscription allows)

Full sync is called every minute or when app is going from background to foreground

Widgets update via /sensor-dense every 30 minutes or when user manually request.
